src/configuration.js

```diff
@@ -50,6 +50,18 @@ const VALID_PROPS = {
 };
 
 /**
+ * All valid user claims, used for validation and filterind .asUser calls
+ * @type {Array}
+ */
+const USER_CLAIMS = ["id", "email", "external_id", "anonymous_id"];
```

Comment:
@michaloo I think we still have connectors that use `guest_id` instead of `anonymous_id`. Let's make sure that we change them too. Maybe add a warning or error if you have guest_id passed in the claims ?